Log-Sources/Tausight/src/parsers/tausight-json.yaml

```
@@ -0,0 +1,80 @@
+  "4747588642892358920", "8232670830360589649", "16213542790019811296", "5496191912874376013",
+  "8808769858715400190", "5096449124245903253", "16991933324974096948", "8036428234314205104",
+  "13890777077388400402", "17547959282144078929", "6321530184547344324", "3130670896678525192",
+  "17972418188646170554", "12731274797778657514", "6364552970100797903", "11942305310378340015",
+  "15344198238245070909", "14324353104253424868", "16440804011291251861", "5728072677546013112",
+  "18417521538414583973", "1527308733729771871", "5765637937236277844", "5725926227810685674",
+  "6976270188366466544", "5620299035547144355", "12062615071481400615", "14038527047231215907",
+  "9658877793219501003", "17189538032147373027", "15804069926237197074", "8150928856379088450",
+  "3392381594439671387", "12449043830770320626", "3884441501437111509", "15289450831590581147",
+  "17457079630649146362", "16762529394326091140", "7967692697909794189", "1977874354450380336",
+  "9926323466519686129", "2068253973091633317", "5372463979363290731", "11524108205031904134",
+  "1311459764055923618", "17367035759339725179", "16614933329223078581", "6229110698169467111",
+  "9544895561993643442", "13312185421629450693", "10734140536465371835", "17110226208560029986",
+  "3233025372120568883", "16070293335899322576", "6146200383242126333", "1990852215035401417",
+  "16211170557513602992", "5632628727331703358", "1687631264888284218"], "executable":
+  {"fileAttributes": ["FILE_ATTRIBUTE_ARCHIVE"], "creationTime": "2019-12-07T09:09:44.320193500Z",
+  "lastAccessTime": "2023-11-15T22:19:04.247864900Z", "companyName": "Microsoft Corporation",
+  "description": "Location Notification", "fileVersion": "10.0.19041.1 (WinBuild.160101.0800)",
+  "fileSize": "71168", "productName": "LocationNotification", "productVersion": "10.0.19041.1"},
+  "signature": {}}, "session": {"authenticationPkg": "Kerberos", "caption": " ", "description":
+  " ", "logonId": 553834, "logonSessionType": "LS_INTERACTIVE"}}, "domain": "BH",
+  "domainAndHost": "BH\\MXL04543WX", "messageId": "4e42dc0dfbdc2dfd4f27aef3ed8d14150211c4088a537fd5748aa3493a7a9976",
+  "eventType": "application_started"}'
+fieldsToBeRemovedBeforeParsing: []
+$schema: https://schemas.humio.com/parser/v0.2.0
+script: "// tausight-json parser for ingesting tausight sensor telemetry\n// Copyright\
+  \ tausight 2023\n\n// Drop events that are not used\ncase {\n @rawstring=\"*moduleInventory*\"\
+  \ or\n @rawstring=\"*driverInventory*\" or\n @rawstring=\"*servicesInventory*\"\
+  \ or\n @rawstring=\"*processInventory*\"\n| dropEvent();\n * }\n\n// Parse as json\
+  \ and exclude unused or arrays with excessive fields that can't be handled\n// default\
+  \ error handling is to drop the first 100 elements and then continue rather than\
+  \ stopping at the 100th.\n| parseJson(exclude=[applicationStarted.mainProcess.modules,\
+  \ tauoneStarted.drivers, tauoneStarted.productList])\n\n// drop events that are\
```

For forwards compatibility all vendor specific fields should be under the `Vendor.` field
```suggestion
  \ tauoneStarted.drivers, tauoneStarted.productList], prefix=\"Vendor.\")\n\n// drop events that are\
```

Bear in mind though that all fields will be prefixed with `Vendor.` so that `fileDeleted.filename` will become `Vendor.fileDeleted.filename`. This does mean that all Dashboards, Alerts, etc will need to be updated, but it is a change that we will have to do at some point. If we choose to postpone this change it will cause a very bad user experience when the same field will have different names in the near future.